update_dcos-kafka_kafka-mesos.jar_to_latest

dmitrypekar opened the pull request:
Hey,

I've just updated dcos-kafka to 0.2.0 and kafka-mesos.jar to 0.9.2.0.
Please merge.

kensipe commented on repo/packages/K/kafka/1/package.json:
```diff
@@ -0,0 +1,28 @@
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Kafka running on top of Apache Mesos",
+  "framework": true,
+  "tags": ["mesosphere", "framework", "bigdata"],
```
we need to remove mesosphere and framework from the tags.  this is used for user search and is intended to differentiate the services.  I would suggest:  "analytics", "queue", "pipeline" anything you imagine a user would search for that you would like kafka to be an answer for... or involved with.

dmitrypekar replied:
Fixed.

kensipe commented on repo/packages/K/kafka/1/marathon.json:
```diff
@@ -0,0 +1,29 @@
+{
+  "id": "{{kafka.app.id}}",
```
can we make this kafka.framework-name ?   the issue is if I install 2 versions of kafka, I am force to change to variables for this to work.  The framework-name and the app.id.

dmitrypekar replied:
Fixed.

kensipe commented on repo/packages/K/kafka/1/marathon.json:
```diff
@@ -0,0 +1,29 @@
+  "id": "{{kafka.app.id}}",
+  "cpus": {{kafka.app.cpus}},
+  "mem": {{kafka.app.mem}},
+  "instances": 1,
```
this needs to be a variable `"instances" : {{kafka.app.instances}}`

dmitrypekar replied:
Fixed.

kensipe commented:
The command-line fails on uninstall... details below:

```
dcos package uninstall kafka
Error while fetching [http://ken-69059-elasticl-ce2322x3nt4a-1645660120.us-west-2.elb.amazonaws.com/mesos/master/shutdown]: HTTP 404: Not Found
```

kensipe commented:
@dmitrypekar sorry for the delay... there looks to be a few things to resolve.

kensipe commented:
while you are at it... either rebase or remove the index.json.gz file

dmitrypekar commented:
@kensipe 
I've rebased PR and fixed following items:
- removed app.id, used framework-name instead;
- added app.instances config param;
- fixed tags;
- retested using local mesos cluster and dcos cli;

Please merge.

kensipe commented:
awesome!  thx... LGTM merging!
